include/openspace/util/timeline.h

```diff
@@ -95,6 +95,16 @@ bool compareTimeWithKeyframeTime(double a, const KeyframeBase& b);
 */
 bool compareKeyframeTimeWithTime(const KeyframeBase& a, double b);
 
+/**
+* Return true if the timestamp of a is smaller than or equal to b.
```

Comment:
The commented out lines should be aligned one character in to make the * of each line align